src/d3a/d3a_core/simulation.py

```diff
@@ -265,7 +265,10 @@ def run(self, resume=False) -> (Period, duration):
                                 self.endpoint_buffer
                             )
 
-                    self.deactivate_areas(self.area)
+                    """
+                    TODO: D3ASIM-1001
+                    """
+                    # self.deactivate_areas(self.area)
```

Comment:
So you decided to remove this? I thought we found the reason somewhere else?

Reply:
There were two different things that were causing energy imbalance after recent merges in master. It is commented and to be solved later in this story (https://gridsingularity.atlassian.net/browse/D3ASIM-1001). The other was due to the merge of branch 965 (https://github.com/gridsingularity/d3a/pull/377).  If you look into this link (https://github.com/gridsingularity/d3a/pull/368/files#diff-19189e4044c6277fcfe2d4bd1b9b99d7R104 & https://github.com/gridsingularity/d3a/pull/368/files#diff-ce55928b3f59db35206112ab5edebc9dR153), trade_rate was incorporated for the partial trade but was overlooked for the whole energy trade.